site/faq/README.md

```diff
@@ -74,6 +74,11 @@ Yes. kpt's architecture facilitates the developing programs which may
 generate or modify configuration. See the [Using Functions] for how to
 compose multiple programs together.
 
+#### **What is the roadmap for kpt?**
```

Comment:
I would move this near the topic. I would order the questions from most important/common to the least. Can be done in a seperate PR.